aodncore/util/misc.py

```diff
@@ -274,6 +275,19 @@ def validate_relative_path(o):
         raise ValueError("path '{o}' must be a relative path".format(o=o))
 
 
+def validate_relative_path_attr(path, path_attr):
+    """Validate a path, raising an exception containing the name of the attribute which failed
+
+    :param path: string containing the path to test
+    :param path_attr: attribute name to include in the exceptions message if validation fails
+    :return: None
+    """
+    try:
+        validate_relative_path(path)
+    except ValueError:
+        raise ValueError("{attr} must be a relative path".format(attr=path_attr))
```

Comment:
That's great, but you could also leave the value of `path` in the error message, just to make debugging easier.